src/popup/locales/en.json

```diff
@@ -1,6 +1,7 @@
 {
   "ok": "OK",
   "ae": "AE",
+  "home": "Home",
```

Comment:
Let's maybe move in into `pages.titles` as other tab bar text

Reply:
Since in the Header component very strange logic around the home title i will improve it and include this into titles.